Kudu.Services.Web/DebugConsole/Default.cshtml

```diff
@@ -1,147 +1,187 @@
-﻿@{

```

Comment:
Do you know what's causing this file do show up as being 100% different here, despite it being mostly the same? Maybe there is a line ending issue with your editor?

I wonder if the fact that we're missing *.cshtml from the .gitattributes contributes to the problem.